contrib/devnet/download_snapshot.py

```
@@ -0,0 +1,118 @@
+SNAPSHOT_CACHE_DIR = HOME_DIR / "zetacored_snapshot_testnet"
+TEMP_EXTRACT_DIR = HOME_DIR / "zetacored_snapshot_temp"
+
+# ============================================================================
+# Helper Functions
+# ============================================================================
+
+def run_command(cmd, shell=True, check=True):
+    """Run a shell command and handle errors."""
+    print(f"Running: {cmd}")
+    try:
+        subprocess.run(cmd, shell=shell, check=check)
```

Ruff (0.14.4)

`subprocess` call: check for execution of untrusted input

(S603)